estate/models/estate_property_offer.py

```diff
@@ -49,4 +54,21 @@ def _compute_deadline(self):
     def _inverse_deadline(self):
         for rec in self:
             # convert this to days, never mind the output is. (it's the date, full date)
-            rec.validity = (rec.deadline - rec.create_date.date()).days
\ No newline at end of file
+            rec.validity = (rec.deadline - rec.create_date.date()).days
+
+
+    def action_accept(self):
+        self.ensure_one()
+        if "accepted" in self.property_id.offer_ids.mapped('status'):
```

Comment:
Le champ devrait s'appeler `state` c'est un nom particulier en Odoo avec des fonctionnalités supplémentaires
```suggestion
        if "accepted" in self.property_id.offer_ids.status:
```

Reply:
esque c'est bon de faire :
if "accepted" in self.property_id.offer_ids.status:

moi je pensais que l'utilisation de mapped() ici c'etait plutot en parcourant tout les recordsets et filtrant que la colonne "status", cette dernier sera retourner sous forme d'un list, donc une truc comme :
['accepted', 'rejected', 'pending']

et cela nouss permet d'avoir la commande suivante :
if "accepted" in ['accepted', 'rejected', 'pending']

sinon je check pour voir si :
"self.property_id.offer_ids.status"

retourne quoi

Reply:
J'ai essayer avec Mapped ca renvoie (dans la console) : 
++++++++++++++ MAPPED : ['accepted', 'refused', False, 'refused']

sans mapped() ca renvoi une erreur

Reply:
@Walter28 C'est moi qui ai fait une erreur avec ce commentaire désolé, tu dois utiliser `mapped` ici

Reply:
@kfr-odoo Merci Kfr 🙏